TUTORIAL.md

```diff
@@ -84,32 +80,33 @@ To setup your Okta account for Spring Boot, you'll first need to create an appli
 
 1. Navigate to your Admin console: i.e. https://dev-123456-admin.oktapreview.com/admin/dashboard
 2. On the top menu click on **Applications**
-3. Click the **Add Application** button
-4. Click the **Create New App** button
-5. On the *Create a New Application Integration* popup fill select the following values, then click the **Create** button
+3. Click **Add Application**
+4. Click **Create New App**
+5. On the *Create a New Application Integration* popup, select the following values, then click **Create**
     - Platform - Native
     - Sign-on Method - OpenID Connect
-6. On the *Create OpenID Connect Integration* page enter the following value, then click the **Next** button
+6. On the *Create OpenID Connect Integration* page, enter the following value, then click **Next**
     - Application Name - Spring Boot Awesomesauce
-7. Use `http://localhost:8080/client/callback` for the Redirect URI's, and press the **Finish** button
+7. Use `http://localhost:8080/client/callback` for the Redirect URI's, and click **Finish**
 
-Your application has been created, but you still have a few settings to change. On the *General* tab, click the **Edit** button on the **General Settings** panel.
+Your application has been created, but you still have a few settings to change. 
 
-1. Check the *Refresh Token*, and *Resource Owner Password* checkboxes and click the **Save** button
-2. Click the **Edit** button on the Client Credentials panel
-3. Select the *Use Client Authentication* radio button, and click the **Save** button
-4. Click on the *Groups* tab
-5. Select the *Assign* button in the *Everyone* column, and click the **Done** button
-6. Grab the ID portion of the URL of your browsers current page, for example: if my URL was: `https://dev-123456-admin.oktapreview.com/admin/app/oidc_client/instance/00icu81200icu812/#tab-groups` then `00icu81200icu812` is your application's ID
+1. On the *General* tab, click **Edit** on the **General Settings** panel
+2. Check the *Refresh Token*, and *Resource Owner Password* checkboxes click **Save**
+3. Click **Edit** on the Client Credentials panel
+4. Select the *Use Client Authentication* radio button, and click the **Save** button 
+5. Click on the *Groups* tab
```

Comment:
This has changed in the Okta UI. I've fixed it in my "tech-review" branch, but we should probably contribute a fix to the Stormpath Java SDK as well.